### PR TITLE
[FIX] hr_attendance: error in log on exporting translation

### PR DESCRIPTION
**Pull request by niyasraphy**
before this commit, on exporting the translation of hr_attendance module shows warning in the log

XML declaration allowed only at the start of the document, line 3, column 6
Opening and ending tag mismatch: t line 4 and templates

after this commit, no warning/error will be shown
in log

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/hr_attendance/static/src/client_action/greeting_message/greeting_message.xml [hunk 1]
```diff
@@ -59,5 +59,5 @@
             </div>
         </t>
     </CardLayout>
-
```
Comment (sofiagvaladze) [inline addons/hr_attendance/static/src/client_action/greeting_message/greeting_message.xml:62]: this is good :)

File: addons/hr_attendance/static/src/client_action/kiosk_confirm/kiosk_confirm.xml [hunk 2]
```diff
@@ -1,5 +1,3 @@
-
```
Comment (sofiagvaladze) [inline addons/hr_attendance/static/src/client_action/kiosk_confirm/kiosk_confirm.xml:1]: this is unnecessary change, especially in stable , to be removed

**Review by sofiagvaladze — CHANGES_REQUESTED** [review]
@niyasraphy, thanks for your contribution :raised_hands: 

The first change where you add `</t>`, but second change where you remove empty lines is unnecessary, especially in stable

Remove the second one and then it can be merged ^^

File: addons/hr_attendance/static/src/client_action/kiosk_confirm/kiosk_confirm.xml (hunk 2 above)
Comment (niyasraphy) [reply addons/hr_attendance/static/src/client_action/kiosk_confirm/kiosk_confirm.xml]: thanks @sofiagvaladze ,
updated the pr

**Comment by sofiagvaladze** [comment]
robodoo r+
